src/box/session.cc

```diff
@@ -262,6 +265,13 @@ session_destroy(struct session *session)
 	mh_i64ptr_remove(session_registry, &node, NULL);
 	credentials_destroy(&session->credentials);
 	sql_session_stmt_hash_erase(session->sql_stmts);
+	struct session *list_session;
+	rlist_foreach_entry(list_session, &active_sessions, in_active_list) {
```

Comment:
Why you need to find this session in list? Why rlist_del_entry(session, in_active_list) is not ok?

Reply:
You are right. Changed to rlist_del_entry.